CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## v0.1.0
+
+This is the first `vm-device` release.
+The `vm-device` crate provides device traits defining read and write operations
+on specialized buses, a device manager for  operating devices and dispatching
```

Comment:
There's an extra space between "for" and "operating".

Reply:
fixed!